code/src/components/AddTodo.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
+import { addTodo } from "reducers/todoData";
+import { Link } from "react-scroll";
+import video from "images/background.mp4"
+const AddTodo = () => {
+  const [addTask, setAddTask] = useState("");
+  const [addTitle, setAddTitle] = useState("");
+
+  const dispatch = useDispatch();
+
+  const onSubmited = () => {
+    if (addTitle === "" || addTask === "") {
+    } else {
+      dispatch(addTodo({ title: addTitle, task: addTask }));
+      setAddTitle('')
+      setAddTask('')
+    }
+  };
+
+
+
+
+
+  return (
+    <article className="addTodoContainer" id="top" >
+        <video id="background-video" autoPlay loop muted>
+    <source src={video} type='video/mp4' />
+</video>
+
+<h1> Add a Todo </h1>
+      <section className="addTodoContent">
+        
+        <section className="addTodo___text">
+       
+          <form>
+            <label>
+              {" "}
+              Title:
+              <input
+                className="titel"
+                type="text"
+                maxLength="25"
+                value={addTitle}
+                onChange={(e) => setAddTitle(e.target.value)}
+              />
+            </label>
+
+            <label>
+              {" "}
+              Task:
+              <textarea
+                className="task"
+                type="text"
+                maxLength="150"
+                value={addTask}
+                onChange={(e) => setAddTask(e.target.value)}
+              />
+            </label>
+          </form>
+
+          <div className="addTodo___btn">
+            <button
+              className={
+                addTitle === "" || addTask === "" ? "btnRed" : "btnGreen"
```

Comment:
Question: why is it named "btnRed" : "btnGreen"? Can't see those colors used in the app?